.github/workflows/actions/download/action.yml

```diff
@@ -27,7 +29,12 @@ runs:
     - name: Download Fuseki release
       shell: bash
       run: |
-        curl -LSfs https://archive.apache.org/dist/jena/binaries/apache-jena-fuseki-4.6.1.tar.gz -o RACK/rack-box/files/fuseki.tar.gz
+        curl -LSfs https://archive.apache.org/dist/jena/binaries/apache-jena-fuseki-${FUSEKI_VERSION}.tar.gz -o RACK/rack-box/files/fuseki.tar.gz
+
+    - name: Download Apache Jena libraries
+      shell: bash
+      run: |
+        curl -LSfs https://archive.apache.org/dist/jena/binaries/apache-jena-${FUSEKI_VERSION}.tar.gz -o RACK/rack-box/files/jena.tar.gz
```

Comment:
You need to tell GitHub CI explicitly when you use an environment variable because run steps can be executed on many shells (even PowerShell).  So you don't write `${FUSEKI_VERSION}`, you write `${{ env.FUSEKI_VERSION }}` instead.  I doubt the env var is worth the effort - it may be easier to just put the 4.6.1 back in (the two lines are close together visually, and nothing should break if the versions accidentally diverge anyway).